contracts/pm/mixins/MixinContractRegistry.sol

```diff
@@ -19,21 +19,21 @@ contract MixinContractRegistry is MContractRegistry, ManagerProxyTarget {
     {}
 
     /**
-     * @dev Returns an instance of the IBondingManager interface
+     * @notice Returns an instance of the IBondingManager interface
```

Comment:
Should these functions use the `@dev` tag?

Reply:
fixed in 8d0bb3e